authentication/views.py

```diff
@@ -73,4 +73,15 @@ def get(
         """
         GET endpoint for logging a user in.
         """
-        return redirect(get_redirect_url(request))
+        redirect_url = get_redirect_url(request)
+        if not request.user.is_anonymous:
+            profile = request.user.profile
+            if (
+                not profile.completed_onboarding
+                and request.GET.get("skip_onboarding", "0") != "1"
```

Comment:
Nit because non-negated comparisons are easier to read, plus it works then for any value that's not a 0, just in case:
```suggestion
                and request.GET.get("skip_onboarding", "0") == "0"
```